shtab/main.py

```diff
@@ -34,12 +34,16 @@ def get_main_parser():
         action="store_true",
         help="raise errors if `parser` is not found in $PYTHONPATH",
     )
+    parser.add_argument("-v", "--verbose", dest="loglevel",
```

Comment:
```suggestion
    parser.add_argument("--verbose", dest="loglevel",
```

to avoid potential confusion with `--version`